### ISSUE TITLE
Accessing Exclusion Files

### ISSUE THREAD

**Issue by pldiguanaman** [GSA/sam_api#111]
I work as a programmer for the Freeman Health System and I am trying to find out if there is a way to download the latest exclusion file programmatically rather than manually downloading.  Not sure if this is the correct forum/location though.  If not can you direct me to where I could find out?

**Comment by aromero6** [comment]
Hi,
If you are currently retrieving the files manually from the SFTP server, you can certainly automate that downloading.  You would just need to write a batch file or cron job to login and retrieve the file.
Thanks,
Tony

**Comment by pldiguanaman** [comment]
We are not manually getting the file via SFTP we are getting the file from the website.   That file is different from the ones available on the SFTP site.  We need a way to get ALL the records in a file (like the web site provides) on the SFTP site in order to facilitate automation.

**Comment by aromero6** [comment]
Can you please send me an email at Antonio.romero@gsa.gov with specifically
which file you download from the website?  We may be able to help you with
this endeavor.

Thanks
Tony


Antonio Romero
US Federal Government
General Services Administration OCIO
Office of Integrated Acquisition Environment
IT Operations Team
O: 312.353.3116
C: 312.497.5592

**Comment by shipras** [comment]
Hello, was anyone able to automate the Exclusion query using API against the latest exclusion database? We're trying to do similar GSA check for our client but want to automate the exclusion search instead of going to website and manually doing it.

**Comment by pldiguanaman** [comment]
No.  I got nowhere asking for this.  I gave up.

**Comment by 0lav** [comment]
>           Hello, was anyone able to automate the Exclusion query using API against the latest exclusion database? We're trying to do similar GSA check for our client but want to automate the exclusion search instead of going to website and manually doing it.

Yes, that is what API's are for... You need to talk to your developer. I made something that queries several names and makes a PDF. It automates a function that was done manually often.
